src/ostorlab/serve_app/types.py

```diff
@@ -463,3 +465,119 @@ def resolve_agents(
 class AgentGroupsType(graphene.ObjectType):
     agent_groups = graphene.List(AgentGroupType, required=True)
     page_info = graphene.Field(common.PageInfo, required=False)
+
+
+class AssetScansMixin:
+    scans = graphene.List(
+        lambda: OxoScanType, last_only=graphene.Boolean(required=False)
+    )
+
+    def resolve_scans(self, info):
+        with models.Database() as session:
+            scans = session.query(models.Scan).filter(
+                models.Vulnerability.asset_id == self.id
```

Comment:
```suggestion
                models.Scan.asset_id == self.id
```

+ i dont see any unit test for this part

Reply:
needs unit test maybe ?

Reply:
Indeed.